### DISCUSSION TITLE
Feature "startAtRow"

### DISCUSSION THREAD

**Discussion by bdzzaid** [firegloves/MemPOI#19] [Ideas]
Hello,

First of all, very good job for the release 1.5.0, bravo ! 

With a delay, I'm trying to complete my migration from Jasper Report to MemPOI. I would like to discuss with you an idea of feature nice to have (in my case). 

Currently with JasperReport, I'm able to set an input as a datasource to render an excel grid after a designed header.
I'm also able to set a row at the end of the grid. Here a printscreen of our process.  

![image](https://user-images.githubusercontent.com/11232571/114874689-d8d3a300-9dfc-11eb-95eb-c6058a7d83d7.png)

The idea is to configure MemPOI to start a process at a given row, instead of starting at the firstRow. 

Given this feature I'm able to do what ever I want in first lines of the excel sheet. 

The last row, unfortunately  ```MempoiSubFooter```. It will be great to have a simple "TextSubFooter" or get a response metadata from MemPOI to get the last written cell position (ie: row: 350, col: 17). 

What do you thing about this feature ?

**Comment by mirkoPan** [comment]
Hi bdzzaid,

thx, it was a pleasure :)

I understand your requirement and it looks like an interesting feature that could be useful to others too.

We are still evaluating the effort and the approach, but after a small talk the more probable release schedule will be to implement (in the following order) :

- start at row
- metadata in the return value
- simple text header
- simple text footer

**Reply by bdzzaid** [reply]
Hi @mirkoPan , 

Thank you for your answers and for your interest to this topic. Let me know if I can help you to implement one of these feature.

Zaïd
